cli/tests/test_global_opts.rs

```diff
@@ -430,7 +430,7 @@ fn test_help() {
       -v, --verbose                      Enable verbose logging
           --color <WHEN>                 When to colorize output (always, never, auto)
           --no-pager                     Disable the pager
-          --config-toml <TOML>           Additional configuration options
+          --config-toml <TOML>           Additional configuration options (can be repeated)
```

Comment:
I'm sure I've commented on some `clap` issue about automatically adding some indication like this (hg uses `[+]`, IIRC) but I can't find it.

Reply:
Maybe one day...